### PR TITLE
added Task2_5

### PR DESCRIPTION
**Pull request by Kr14va**

### CODE REVIEW

**Comment by Kr14va** [comment]
[Задания №2](https://trello.com/c/0fjOdxti/94-%D0%B7%D0%B0%D0%B4%D0%B0%D0%BD%D0%B8%D1%8F-%E2%84%962)

File: Task2_5/Task5/Program.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
+﻿using System;
+
+namespace Task5
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // Треугольник ABC. Стороны AB, BC, AC
+
```
Comment (PaShkadun) [inline Task2_5/Task5/Program.cs:10]: No whiteline.

File: Task2_5/Task5/Program.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            Console.WriteLine("Введите длинну стороны AB: ");
+            int sideAB = Convert.ToInt32(Console.ReadLine());
```
Comment (PaShkadun) [inline Task2_5/Task5/Program.cs:12]: Without exception.
Without two upper chars.

File: Task2_5/Task5/Program.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+            Console.WriteLine("Введите длинну стороны BC: ");
+            int sideBC = Convert.ToInt32(Console.ReadLine());
+
+            Console.WriteLine("Введите длинну стороны AC: ");
+            int sideAC = Convert.ToInt32(Console.ReadLine());
+
+            // Треугольник сущестует только в том случае, если сумма двух сторон не меньше третьей стороны
+
```
Comment (PaShkadun) [inline Task2_5/Task5/Program.cs:21]: No whiteline.

File: Task2_5/Task5/Program.cs [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            if (sideAB + sideBC >= sideAC && sideBC + sideAC >= sideAB && sideAB + sideAC >= sideBC)
+            {
+                Console.WriteLine("Такой треугольник существует!");
+                if (sideAB == sideBC && sideBC == sideAC && sideAC == sideAB)
```
Comment (PaShkadun) [inline Task2_5/Task5/Program.cs:25]: Whiteline.

File: Task2_5/Task5/Program.cs [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+                {
+                    Console.WriteLine("Это равносторонний треугольник!");
+                }
+
```
Comment (PaShkadun) [inline Task2_5/Task5/Program.cs:29]: No whiteline.

File: Task2_5/Task5/Program.cs [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+                else if (sideAB == sideBC || sideAC == sideAB || sideBC == sideAC)
+                {
+                    Console.WriteLine("Это равнобедренный треугольник!");
+                }
+            }
+            else
+            {
+                Console.WriteLine("Такой треугольник не существует!");
+            } 
+                Console.ReadKey();
```
Comment (PaShkadun) [inline Task2_5/Task5/Program.cs:39]: wtf?

**Review by PaShkadun — CHANGES_REQUESTED** [review]

**Pull request closed without merging**